include/mapnik/raster.hpp

```diff
@@ -40,15 +40,27 @@ class raster : private util::noncopyable
 {
 public:
     box2d<double> ext_;
+    box2d<double> query_ext_;
```

Comment:
@flippmoke - can we get away with just passing x/y offsets e.g `point<double>` ?